.build/build-resolver.xml

```diff
@@ -212,6 +212,26 @@
         </retry>
         <mkdir dir="${local.repository}/org/apache/cassandra/deps/sigar-bin"/>
         <mkdir dir="${build.lib}/sigar-bin"/>
+        <mkdir dir="${build.lib}/x86_64"/>
+        <mkdir dir="${build.lib}/aarch_64"/>
```

Comment:
This should be changed to `aarch64` because that is the output of `uname -m` we try to match. I have a commit for that but it is not pushed yet because pushing would cancel the build in Circle I am running right now. Will do later.